test/05_variable/variable.py

```diff
@@ -0,0 +1,25 @@
+import unittest
+
+try:
+    import variable_pybind11.nested_namespace as py11
```

Comment:
I may have already asked this, but how do we get these top-level namespaces per each binding type?